### PR TITLE
IIB timeout changed to 1hr and pipeline timeout changed to 1.5hr

### PR DESCRIPTION
**Pull request by mitul564**
Signed-off-by: Mitulsharma123 <sharmamitul72@gmail.com>

### CODE REVIEW

**Comment by mitul564** [comment]
@amisstea Can you validate the changes and ignore/delete/close past PRs (#265 #269 #256)

File: operator-pipeline-images/operatorcert/entrypoints/index.py [hunk 1]
```diff
@@ -124,7 +124,8 @@ def publish_bundle(
     resp = iib.add_builds(iib_url, payload)
 
     batch_id = resp[0]["batch"]
-    response = wait_for_results(iib_url, batch_id)
+    delay = 3600  #60*60
+    response = wait_for_results(iib_url, batch_id, delay)
```
Comment (amisstea) [inline operator-pipeline-images/operatorcert/entrypoints/index.py:128]: Please change the default value for the `timeout` kwarg on `wait_for_results` instead.

File: ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml [hunk 2]
```diff
@@ -81,7 +81,7 @@
                 metadata:
                   generateName: operator-hosted-pipeline-run
                 spec:
-                  timeout: "2h"
+                  timeout: "1.5h"
```
Comment (amisstea) [inline ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml:84]: This is changing the timeout on the wrong pipeline. Please add the 1.5h timeout to the release pipeline trigger instead.

**Review by amisstea — CHANGES_REQUESTED** [review]

File: ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml (hunk 2 above)
Comment (mitul564) [reply ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml:84]: https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/operator-pipeline-images/tests/data/pipelinerun.json#L332

Are below changes correct ?
`"timeout": "1h0m0s", to "timeout": "1h30m0s",`

File: operator-pipeline-images/operatorcert/entrypoints/index.py (hunk 1 above)
Comment (mitul564) [reply operator-pipeline-images/operatorcert/entrypoints/index.py:128]: https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/operator-pipeline-images/operatorcert/entrypoints/index.py#L44

Looks Good ? @amisstea 

```
New:
line 44:
def wait_for_results(iib_url: str, batch_id: int, timeout=60 * 60, delay=20) -> Any:
line 51:
timeout ([type], optional): Maximum wait time. Defaults to 60*60 (3600 seconds/1 hour).
```

File: ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml (hunk 2 above)
Comment (amisstea) [reply ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml:84]: It needs to be added here: https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/a9f8286bdf42ad4a01312f725cd55d1f7c10d114/ansible/roles/operator-pipeline/tasks/webhook-release-pipeline-trigger.yml#L65

File: operator-pipeline-images/operatorcert/entrypoints/index.py (hunk 1 above)
Comment (amisstea) [reply operator-pipeline-images/operatorcert/entrypoints/index.py:128]: That's the correct location. Please push your changes so they can be reviewed.

**Comment by mitul564** [comment]
@amisstea Done, please have a look ! 

https://github.com/redhat-openshift-ecosystem/operator-pipelines/pull/266/commits/c17d99c877fe521b44023256ece4322b158bd717

**Review by amisstea — APPROVED** [review]

**Pull request merged**